packages/frontend/src/pages/insight-editor/insight-draft-container.tsx

```diff
@@ -127,7 +127,7 @@ export const InsightDraftContainer = ({ insight, draft, draftKey, onRefresh }) =
       });
     }
 
-    return data;
+    return data!;
```

Comment:
This is disabled in the `frontend` package so you can't use it.

If data can be undefined we should change the return type to `Promise<UploadSingleFileMutation | undefined>`